client/run.go

```diff
@@ -136,19 +86,100 @@ func (cli *HyperClient) HyperCmdRun(args ...string) (err error) {
+		Name          string   `long:"name" value-name:"\"\"" description:"Assign a name to the container"`
+		Attach        bool     `short:"a" long:"attach" default:"false" default-mask:"-" description:"(from podfile) Attach the stdin, stdout and stderr to the container"`
+		Detach        bool     `short:"d" long:"detach" default:"false" default-mask:"-" description:"(from cmdline) Not Attach the stdin, stdout and stderr to the container"`
+		Workdir       string   `long:"workdir" value-name:"\"\"" default-mask:"-" description:"Working directory inside the container"`
+		Tty           bool     `short:"t" long:"tty" default:"false" default-mask:"-" description:"the run command in tty, such as bash shell"`
+		Cpu           int      `long:"cpu" default:"1" value-name:"1" default-mask:"-" description:"CPU number for the VM"`
+		Memory        int      `long:"memory" default:"128" value-name:"128" default-mask:"-" description:"Memory size (MB) for the VM"`
+		Env           []string `long:"env" value-name:"[]" default-mask:"-" description:"Set environment variables"`
+		EntryPoint    string   `long:"entrypoint" value-name:"\"\"" default-mask:"-" description:"Overwrite the default ENTRYPOINT of the image"`
+		RestartPolicy string   `long:"restart" default:"never" value-name:"\"\"" default-mask:"-" description:"Restart policy to apply when a container exits (never, onFailure, always)"`
+		LogDriver     string   `long:"log-driver" value-name:"\"\"" description:"Logging driver for Pod"`
+		LogOpts       []string `long:"log-opt" description:"Log driver options"`
+		Remove        bool     `long:"rm" default:"false" default-mask:"-" description:"Automatically remove the pod when it exits"`
+		Portmap       []string `long:"publish" value-name:"[]" default-mask:"-" description:"Publish a container's port to the host, format: --publish [tcp/udp:]hostPort:containerPort"`
+		Labels        []string `long:"label" value-name:"[]" default-mask:"-" description:"Add labels for Pod, format: --label key=value"`
+		Volumes       []string `short:"v" long:"volume" value-name:"[]" default-mask:"-" description:"Mount host file/directory as a data file/volume, format: -v|--volume=[[hostDir:]containerDir[:options]]"`
+	}
+
+	var (
+		PodId    string
+		specJson string
+		attach   bool = false
+		err      error
+	)
+	var parser = gflag.NewParser(&opts, gflag.Default|gflag.IgnoreUnknown|gflag.PassAfterNonOption)
+	parser.Usage = fmt.Sprintf("%s [OPTIONS] IMAGE [COMMAND] [ARG...]\n\nCreate a pod, and launch a new VM to run the pod", cmd)
+	args, err = parser.ParseArgs(args)
+	if err != nil {
+		if !strings.Contains(err.Error(), "Usage") {
+			return nil, err
+		} else {
+			return nil, nil
+		}
+	}
+
+	if opts.Container {
+		if len(args) == 0 {
+			return nil, fmt.Errorf("%s: \"%s\" requires the pod id as first argument.", os.Args[0], cmd)
+		}
+		PodId = args[0]
```

Comment:
Here will lose image, the command line format is `hyperctl create [options] IMAGE [COMMAND] ARGS`